ocfweb/docs/templates/docs/partials/account.md

```diff
@@ -0,0 +1,37 @@
+What Your OCF Account Gets You
+
+Many think of the Open Computing Facility as an on-campus resource for free printing. While this is true, and printing is certainly our most-demanded service (we print hundreds of thousands of pages per year), there are plenty of other great services provided by the OCF, which come free with your account!
+
+## Free Computing
+
+The OCF was founded to provide all members of the Cal campus community with computing resources. Anytime we're open, all members of the community are welcome to use the computers for any reason – not just to print. All of the computers in the OCF run Linux as the operating system and have a variety of useful (although potentially unfamiliar) applications installed. Because of our committment to free, open-source software, we do not provide access to application suites such as Microsoft Office or Adobe Creative Cloud; instead, we provide free alternatives.
```

Comment:
Can strike "as the operating system" and "(although potentially unfamiliar)" here.

I also think it's a bit disingenuous to invoke our commitment to software freedom here, since we include plenty of nonfree software on the desktops (including Steam, which you should mention in the list). I would remove the last sentence entirely, since nonfree alternatives get discussed later in the list.

Reply:
Agreed, I would just remove this line.